db/migrate/20240111000939_create_comments.rb

```diff
@@ -0,0 +1,15 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.string :text
```

Comment:
- [ ] Hello, awesome work so far! 🌟 To enhance user flexibility in inputting longer text without running into database limitations, consider changing the data type for the "text" column from string to the more accommodating text type in your Rails

```suggestion
      t.text :text
```